hail/src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -328,7 +328,7 @@ class FunctionBuilder[F >: Null](val parameterTypeInfo: Array[MaybeGenericTypeIn
       cn.accept(cw)
       bytes = cw.toByteArray
 //       This next line should always be commented out!
-//      CheckClassAdapter.verify(new ClassReader(bytes), false, new PrintWriter(sw1))
+      CheckClassAdapter.verify(new ClassReader(bytes), false, new PrintWriter(sw1))
```

Comment:
This next line should be commented out!

Reply:
This needs a rebase I think. Not sure why the diff hasn’t included the changes from the stacked pr

Reply:
Ohh, probably the stacked label blocks.

Reply:
But the above comment should still be removed?